CONTRIBUTING.rst

```diff
@@ -204,39 +204,3 @@ Developing with SLDS
 MetaCI uses https://github.com/SalesforceFoundation/django-slds which imports version 2.1.2 of the Salesforce Lightning Design System.
 
 You can find a CSS and component reference archived here: https://archive-2_1_2.lightningdesignsystem.com/
-
```

Comment:
What's the story behind this deletion?

Reply:
@prescod my changes to the populate_db script (mentioned above in this doc) now take care of doing these steps automatically.